dev-python/slixmpp/slixmpp-1.5.2.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_7 )
```

Comment:
Can you try tests with python 3.8, 3.9 too?

Reply:
Tested with 3.8, some dependencies of slixmpp and poezio do not declare compatibility with python 3.9 (yet).

Reply:
Can you file bugs for those dependencies if they don't already exist?